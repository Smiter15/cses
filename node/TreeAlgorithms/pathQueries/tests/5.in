10 10
4 4 1 4 1 9 5 9 10 5
1 10
5 2
3 2
8 6
4 9
4 1
4 5
3 6
7 5
1 9 3
1 2 7
2 1
2 9
2 7
1 6 10
2 5
1 9 5
2 5
1 3 4
