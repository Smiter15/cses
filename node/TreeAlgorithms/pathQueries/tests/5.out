4
11
14
9
9
